fabric-renderer-registries-v1/src/main/java/net/fabricmc/fabric/api/client/rendereregistry/v1/EntityRendererRegistry.java

```diff
@@ -26,7 +26,10 @@
 
 /**
  * Helper class for registering EntityRenderers.
+ *
+ * @deprecated This module has been moved into fabric-rendering-v1. Use {@link net.fabricmc.fabric.api.client.rendering.v1.EntityRendererRegistry} instead
  */
+@Deprecated(forRemoval = true)
 public interface EntityRendererRegistry {
 	EntityRendererRegistry INSTANCE = new EntityRendererRegistryImpl();
```

Comment:
may as well define this inline (anonymous class?), same in BlockEntityRendererRegistry